calcom/cal.com
# Fixing GCal emailing notif and sequential deleting

## What does this PR do?

Making sure Google Calendar does not send deleted events notifications when deleted by API, as we handle the notification for them. Also, Google Calendar delete was causing problems when using its API in parallel, changing it to be sequential.

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Try to cancel a recurring event with lots of instances and you shouldn't see Google sending cancelled notification for each instance nor a 500 error when doing so.
